parser/src/parser.cpp

```diff
@@ -204,16 +213,32 @@ int main(int argc, char* argv[])
     return 1;
   }
 
-  //--- Check workspace and project directory ---//
+  //--- Check database and project directory existence ---//
+
+  bool isNewDb = cc::util::connectDatabase(
```

Comment:
`!cc::util::connectDatabase(...)`?

Reply:
The whole instruction goes like this:
```cpp
bool isNewDb = cc::util::connectDatabase(
  vm["database"].as<std::string>(), false) ? false : true;
```
which is basically the same. Could be simplified to your suggestion, I just didn't want to negate the returned shared pointer, as that would seem like a boolean value is returned and could be confusing.

Reply:
This isn't fixed yet. Still, the tertiary operator feels too verbose here. How would you like the following as a compromise?

```cpp
bool isNewDb = cc::util::...(...) == false;
```

Reply:
In that case I think checking the equality against `nullptr` would still be better, otherwise it would still ambiguously imply that `connectDatabase()` returns with a boolean value.

Reply:
Fixed in 1c4b8809a7139dc3519037d42d650b2e0e339686.